packages/edit-navigation/src/components/menu-editor/style.scss

```diff
@@ -68,6 +68,10 @@
 	-ms-grid-row-align: start;
 }
 
+.edit-navigation-menu-editor__cancel-create-menu-button {
+	margin-left: 1ch;
```

Comment:
It seems like we're using either 6px or 8px for this kind of margin in other places.